app/components/global/layout/SideNavLayout.js

```diff
@@ -0,0 +1,50 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import styled from 'styled-components'
+import { NavLink } from 'react-router-dom'
+import { Flex, Box } from 'grid-styled'
+import { th } from '@pubsweet/ui-toolkit'
+
+const SideNavLink = styled(NavLink)`
```

Comment:
Can I suggest that we pull a generic `NavLink` styled in the eLife way out into `atoms`?

Then we can use it here and [in `AppBar`](https://github.com/elifesciences/elife-xpub/blob/develop/app/components/global/AppBar/AppBar.js#L17-L28) (albeit with ever so slightly different styles applied in each situation).

Partly just to reduce duplication. But also because, having it in `atoms` then makes it more obvious to anyone wanting to use a `NavLink` again, that we already have a pattern designed for this.